README.md

```diff
@@ -24,6 +24,14 @@ Before building and running any of the inner projects, you should download the s
 git submodule update --init --recursive
 ```
 
+### Generating files
+
+You can run the Makefile in the projet to generate the project files auto-generated files, simply run the command at the root:
```

Comment:
Please explain we commit the generated files and this step is only required when changing them.